.github/workflows/release.yml

```diff
@@ -21,6 +21,7 @@ jobs:
   continuous-integration:
     uses: ./.github/workflows/continuous-integration.yml
     name: continuous integration
+    secrets: inherit
```

Comment:
Can we pass just the codecov token to avoid unintentionally passing release-only secrets to this workflow?

Reply:
Yep